### PR TITLE
Prepare v2.6

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge.turrado@docplanner.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,39 +24,53 @@
...
-- **General:**  fail fast on `buildScalers` when not able to resolve a secret that a deployment is relying on ([#2394](https://github.com/kedacore/keda/pull/2394))
-- **CPU Scaler:** Adding e2e test for the cpu scaler ([#2441](https://github.com/kedacore/keda/pull/2441))
-- **AWS SQS Scaler:** allow using simple queue name instead of URL ([#2457](https://github.com/kedacore/keda/pull/2457))
-- **AWS Scalers:** Support temporary AWS credentials using session tokens ([#2573](https://github.com/kedacore/keda/pull/2573))
 - **Prometheus Scaler:** Support namespaced Prometheus queries ([#2575](https://github.com/kedacore/keda/issues/2575))
 
 ### Breaking Changes
 
-- No longer push to Docker Hub as of v2.5 as per our [announcement in March 2021](https://github.com/kedacore/keda/discussions/1700).
-  - Learn more about the background on [kedacore/governance#16](https://github.com/kedacore/governance/issues/16)
```
Comment (zroubalik) [inline CHANGELOG.md:54]: Keep it this way, this is one item, with subitem as an explanation.

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,39 +24,53 @@
...
 ### Improvements
 
-- **General:** `keda-operator` Cluster Role: add `list` and `watch` access to service accounts ([#2406](https://github.com/kedacore/keda/pull/2406))|([#2410](https://github.com/kedacore/keda/pull/2410))
+- **AWS Scalers:** Support temporary AWS credentials using session tokens ([#2573](https://github.com/kedacore/keda/pull/2573))
+- **AWS SQS Scaler:** Allow using simple queue name instead of URL ([#2457](https://github.com/kedacore/keda/pull/2457))
+- **Azure EventHub Scaler:** Don't expose connection string in metricName ([#2404](https://github.com/kedacore/keda/pull/2404))
+- **Azure Pipelines Scaler:** Support `poolName` or `poolID` validation ([#2370](https://github.com/kedacore/keda/pull/2370))
+- **CPU Scaler:** AAding e2e test for the cpu scaler ([#2441](https://github.com/kedacore/keda/pull/2441))
+- **External Scaler:** Fix wrong calculation of retry backoff duration ([#2416](https://github.com/kedacore/keda/pull/2416))
 - **General:** Delete the cache entry when a ScaledObject is deleted ([#2564](https://github.com/kedacore/keda/pull/2564))
```
Comment (zroubalik) [inline CHANGELOG.md:59]: I think that it would be better if we put `General` to the beginning or end of the list.

**Review by zroubalik — CHANGES_REQUESTED** [review]
nits

File: CHANGELOG.md (hunk 2 above)
Comment (tomkerkhove) [reply CHANGELOG.md:59]: Agreed, that was my intention with the PR indeed

File: CHANGELOG.md (hunk 1 above)
Comment (tomkerkhove) [reply CHANGELOG.md:54]: Yes, this was the intention indeed

File: CHANGELOG.md (hunk 2 above)
Comment (JorTurFer) [reply CHANGELOG.md:59]: So general at the top, and behind it the other alphabetically sorted, do you agree?

**Review by tomkerkhove — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]

**Pull request merged**